scripts/import_certificates.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
+#
+# Copyright (C) 2020-2021 Bosch.IO GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+# License-Filename: LICENSE
+
+CERTIFICATE_FILES=$1
+
+if [ -z "$CERTIFICATE_FILES" ]; then
+  echo "No certificates specified. Skip installation."
+  exit 0
+fi
+
+echo "Installing certificates $CERTIFICATE_FILES"
```

Comment:
s/Installing/Importing/

Either dot at end of sentence, or ":" after "certificates".